src/raster/r.forcircular/r.forcircular.html

```diff
@@ -0,0 +1,271 @@
+dist_max_fw=900. <br />
+slp_max_cop=30. <br />
+dist_max_cop=800. <br />
+hf_slope="cable crane - high power" <br />
+c_slope="cable crane - medium/low power" <br />
+hf_noslope="tractor" <br />
+c_noslope="tractor" <br />
+ind1=indicator1 <br />
+ind2=indicator2 <br />
+ind3=indicator3 <br />
+ind4=indicator4 <br />
+ind5=indicator5 <br />
+ind6=indicator6 <br />
+ind7=indicator7 <br />
+<br />
+The output maps are: technical_surface (forest surface where exploitation is
+technically possibile), economic_surface (forest surface where the exploitation
+is economically advantageous), emission (co<sub>2</sub> produced by the
+exploitation), annual_avoided_emission (emissions avoided compared to the use of
+fossil fuels), revenues (map of economic revenues), dip (distance from ideal,
+the assessment of the circual economy level of the forest supply chain).
+<br><br>
+Below there are two example of output map corresponding respectively to the
+stumpage value and the annual avoided emission: <br><br>
+<div style="float: left;">
+    <img src="./stumpage.png" alt="">
+</div>
+<div style="float: left;">
+    <img src="./avoided_emission.png" alt="">
+</div>
+<div style="clear:both"></div>
+The output text results are the follow: <br><br>
+############################# <br />
+End of process <br />
+Name of output maps: <br />
+rep_roundwood -> roundwood (m3/y): 8507.3917<br />
+rep_timber -> timber pole (m3/y): 195.4894<br />
+rep_firewood -> firewood (m3/y): 7239.1150<br />
+rep_bioenergy -> bioenergy (MWh/y): 6354.3441<br />
+rep_sum_a_stumpage_value -> annual stumpage value (EUR/y): 147608.5463<br />
+rep_ave_stumpage_value -> average stumpage value (EUR/ha): 3998.2780<br />
+rep_ave_a_stumpage_value -> average annual stumpage value (EUR/ha*y-1):
+28.2872<br />
+rep_annual_avoided_emission -> annual avoided emissions (t): 1963.9961<br />
+---------------------------<br />
+rep_indicator1 -> annual value of wood on annual yield (euro/m3): 7.2534<br />
+rep_indicator2 -> carbon dioxide emission per cubic meter (t/m3): 0.0001<br />
+rep_indicator3 -> general index of forest surface utilization (ha/y):
+157.192<br />
+8
+rep_indicator4 -> general index of re-use (m3*y) sum: 292481.3438<br />
+rep_indicator5 -> potential value of wood on real value (euro/euro):
+1.3093<br />
+rep_indicator6 -> percentual of wood residuals used in bioenergy production (%):
+0.1277<br />
+rep_indicator7 -> avoided CO2 per unit of energy produced (gCO2 /kWh):
+308.6331<br />
+dist_tot -> AMC map: 3.7276<br />
+#############################<br /><br />
+
+
+<h2>REFERENCE</h2>
+Paletto A, Becagli C, Geri F, Sacchelli S, De Meo I. Use of Participatory
+Processes in Wood Residue Management from a Circular Bioeconomy Perspective: An
+Approach Adopted in Italy. Energies. 2022; 15(3):1011.
+https://doi.org/10.3390/en15031011
+
+<h2>SEE ALSO</h2>
+<em>
+    <a href="r.green.biomassfor.html">r.green.biomassfor</a>
+</em>
+
+<h2>AUTHORS</h2>
+Francesco Geri and Sandro Sacchelli
```

Comment:
Good practice to end files with a newline (empty last line):
```suggestion
Francesco Geri and Sandro Sacchelli

```